docs/topics/proc_deploying-a-node-js-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,45 @@
+[id='deploying-a-node-js-application-to-standalone-rhel_{context}']
+= Deploying a {Node} application to standalone Red Hat Enterprise Linux
+
+Deploy a {Node} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A {Node} application.
+* npm 6.4.1 installed
+* RHEL 7 installed.
+* {Node} installed
+x
+
+.Procedure
+
+. Navigate to the directory with your application project:
++
+[source,bash,subs="attributes+",options="nowrap"]
+----
+$ cd myApp
+----
+
+. If you have specified additional dependencies in the `package.json` file of your project, ensure that you install them before running your applications:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install
+----
+
+. Start your application.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ node app.js
```

Comment:
Depending on the application, there could be a number of different ways to start a Node app. This method assumes the entry point to the application is `app.js`. That may or may not be the case. It could be `index.js` or `www/app.js` or any other thing. Typically, though, best practice would be to have a start script in your package.json. In that case, the command would be `npm start`.

Reply:
I think it would be preferable to have an example of both `package json` and `app.js` showing where thew entrypoint and starter script are set.
(Due to time constraints we are currently facing resulting form the imminent closing of train G, I'm not sure we're able to deliver all of these changes int his PR, but I would definitely like to make them in a follow up PR, if possible.)